### PR TITLE
Load dev apps

### PR DESCRIPTION
**Pull request by ksofa2**

### CODE REVIEW

File: apps_dev/dev-entrypoint.sh [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 set -e
 
-echo $(pwd)
-for app in $(ls -d ../../apps_dev/*/); do
-    pip install -e ${app} || echo "couldn't install ${app}"
+cd ../..
+
+for app in $(ls -d apps_dev/*/); do
+    pip install -e ${app} || echo "$(basename ${0}): couldn't install ${app}"
```
Comment (rbharadwaj9) [inline apps_dev/dev-entrypoint.sh]: This should be `pip3 install` since the system-wide pipenv installation doesn't run till after this entrypoint script.

**Comment by ksofa2** [comment]
@rbharadwaj9 Is that necessary? We are running in an image that's built from python:3.7

**Comment by rbharadwaj9** [comment]
Right now pip and python without pipenv install doesn't use python 3. If you run bash inside the container it's evident. However I never realized that it shouldn't be the case since the image is built on python:3.7 
If you want I can look into it and find out what's wrong.

**Comment by ksofa2** [comment]
Oh yeah, I see. I'll change it.

**Comment by ksofa2** [comment]
@jlost What do you think of these changes?

**Comment by rbharadwaj9** [comment]
I was looking into yesterday's pip/pip3 issues and the script works with pip now. I didn't make any specific changes to tackle this issue but does this mean we don't need it to be pip3 anymore?

**Comment by ksofa2** [comment]
It's fine to leave with the change. Good to be explicit about python3 and
related commands.

**Pull request merged**